### PR TITLE
Update about TAA for c18

### PR DESCRIPTION
**Pull request by kyra-at-ada**
**Summary of Changes**
- Remove outdated requirements from About TAA
- Update links for C18
- Add submission link for study plan

Note: Some links are still outdated. Need CSM input before updating

### CODE REVIEW

File: 00-about-thursdays-at-ada/norms.md [hunk 1]
```diff
@@ -1,19 +1,15 @@
 # Thursdays at Ada Norms
 
-* **Attendance at Thursdays at Ada is required for graduation** from the program. 
-  * We expect interns to be present for the entirety of the four hour Thursdays at Ada sessions. 
-  * We understand that sometimes things come up and external circumstances may cause you to miss part of or the entirety of a session. If you need to miss a CS Fundamentals or Career Services session, please send an email to [absent_internship@adadevelopersacademy.org
-  ](mailto:absent_internship@adadevelopersacademy.org
-  ) in advance when possible.
-  * Up to two excused absences are permitted throughout the course of Thursdays at Ada. This is across both Career Development and CS Fundamentals sessions.
+* **Attendance at Thursdays at Ada is optional for graduation** from the program. 
+  * We expect interns to be choose to engage with live Thursdays at Ada content as is most helpful to their learning needs.
```
Comment (char-adadev) [inline 00-about-thursdays-at-ada/norms.md:4]:
```suggestion
  * We expect interns to choose to engage with live Thursdays at Ada content as is most helpful to their learning needs.
```

File: 00-about-thursdays-at-ada/norms.md [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
 * CS Fundamentals assignments are a required part of your continuing education and graduation requirements. 
-  * **You need to complete 80% of all assigned work for CS Fundamentals**. 
-  * New topics and problem sets are released two weeks before they are due
+  * **You need to complete 80% of problem sets for CS Fundamentals**.
```
Comment (char-adadev) [inline 00-about-thursdays-at-ada/norms.md:6]:
```suggestion
  * **You need to complete 80% (4/5) of problem sets for CS Fundamentals**. 
```

File: 00-about-thursdays-at-ada/norms.md [hunk 3]
```diff
@@ -1,19 +1,15 @@
...
-  ](mailto:absent_internship@adadevelopersacademy.org
-  ) in advance when possible.
-  * Up to two excused absences are permitted throughout the course of Thursdays at Ada. This is across both Career Development and CS Fundamentals sessions.
+* **Attendance at Thursdays at Ada is optional for graduation** from the program. 
+  * We expect interns to be choose to engage with live Thursdays at Ada content as is most helpful to their learning needs.
 * CS Fundamentals assignments are a required part of your continuing education and graduation requirements. 
-  * **You need to complete 80% of all assigned work for CS Fundamentals**. 
-  * New topics and problem sets are released two weeks before they are due
+  * **You need to complete 80% of problem sets for CS Fundamentals**. 
+  * **You need to complete 100% of Interview Practice Questions for CS Fundamentals**.
```
Comment (char-adadev) [inline 00-about-thursdays-at-ada/norms.md:7]:
```suggestion
  * **You need to complete 100% (5/5) of Interview Practice Questions for CS Fundamentals**.
```

File: 00-about-thursdays-at-ada/norms.md [hunk 4]
```diff
@@ -1,19 +1,15 @@
...
+  * **You need to complete 5 peer-to-peer mock interviews for CS Fundamentals**.
+  * New topics and problem sets are released at least two weeks before they are due
   * New projects are released four weeks before they are due
```
Comment (char-adadev) [inline 00-about-thursdays-at-ada/norms.md:10]: We might want to update this or delete as we don't have any standardized "due dates" for this cohort.

File: 00-about-thursdays-at-ada/about-thursdays-at-ada.md [hunk 5]
```diff
@@ -3,15 +3,15 @@
...
 - [Graphs: BFS & DFS](../04-graphs/01-graphs.md)
-- [Graphs: Dijkstra](../05-dijkstras/01-graphs-review.md)Programming & Greedy Algorithms
+- [Graphs: Dijkstra](../05-dijkstras/01-graphs-review.md)
 
 ### Reverse Classroom
 
-CS Fundamentals is taught in a [flipped-classroom](https://omerad.msu.edu/index.php?option=com_content&view=article&id=162:what-why-and-how-to-implement-a-flipped-classroom-model&catid=27:teaching) style. Students are assigned a week prior to review a markdown lesson with accompanying video lesson and slides. In class students are expected to ask questions, review concepts, and work on exercises in groups. Students are also given time to engage in a mock interview with a peer.
+CS Fundamentals is taught in a [flipped-classroom](https://omerad.msu.edu/index.php?option=com_content&view=article&id=162:what-why-and-how-to-implement-a-flipped-classroom-model&catid=27:teaching) style. Students are provided with a Learn topic to review two weeks prior to a live review session. In class students are expected to ask questions, review concepts, and work on exercises in groups. 
 
 ### CS Fun Recorded Sessions
```
Comment (char-adadev) [inline 00-about-thursdays-at-ada/about-thursdays-at-ada.md:16]: I think we'll need to update these links/create the Panopto folders for them. I added a task in Asana for us to track: https://app.asana.com/0/1201700438643002/1203988207485365/f

File: 00-study-plan/study-plan.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+ Thursdays at Ada deadlines are flexible. We recognize that each intern will have a different internship experience where some may experience greater intensity at the beginning of internship, others may experience a heavier workload at the end of internship, and others may experience something in between.
+
+ To ensure that you recognize the scope of work we are asking you to complete and have a plan in place to complete the Thursdays at Ada course, we ask that you submit a study plan with proposed completion dates for each of the required elements.
+ 
+ See more details about each plan in your section specific study guide:
+ - [Ligers TAA Study Plan Guide](https://docs.google.com/document/d/19RsoW-SDkUapg0CQuTbCLKaUHuPGDFs6jh85QsjSTJs/edit?usp=sharing)
+ - [Tigons TAA Study Plan Guide](https://docs.google.com/document/d/1pwa85WrRYpqs6rVNPSqOB6QZo-BH4tQcItQ_k99gSkc/edit?usp=sharing)
+
+ ## Completion Options
+There a three basic paths to complete Thursdays at Ada requirements.
```
Comment (char-adadev) [inline 00-study-plan/study-plan.md:12]:
```suggestion
There are three basic paths to complete Thursdays at Ada requirements.
```

File: 00-study-plan/study-plan.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+If you are able to read through the new topic material (without completing the problem set questions) during your first two months of internship , we would highly recommend you do so that the topics are not completely unfamiliar when you start intensive work on CS Fundamentals material.
+
+ ### Path C
+ Choose your own completion dates by filling out and submitting [your personal study plan](https://docs.google.com/document/d/1uCUKu9sZLUSbxsUVobnf8d2EaudXIQxqFpSRhhYE2jU/edit?usp=sharing)
+
+ ## Need to Change your Study Plan?
+ We expect that most interns will need to adjust their intial study plan . If you are altering any milestones we ask that you update your study plan and fill out the milestone extension form, to give the Thursdays at Ada team awareness of your progress:
```
Comment (char-adadev) [inline 00-study-plan/study-plan.md:32]:
```suggestion
We expect that most interns will need to adjust their initial study plan. If you are altering any milestones we ask that you update your study plan and fill out the milestone extension form, to give the Thursdays at Ada team awareness of your progress:
```

**Review by char-adadev — APPROVED** [review]
Looks great, thank you for updating these docs! I left a few suggestions to address some typos but overall everything matched what I would expect 👍🏾

File: 00-about-thursdays-at-ada/about-thursdays-at-ada.md (hunk 5 above)
Comment (kyra-at-ada) [reply 00-about-thursdays-at-ada/about-thursdays-at-ada.md:16]: Yeah I think right now everything is in a general folder. I was going to ask Sylvia if there's a standard organization method we should be using on Tuesday.

**Pull request merged**